Source/WebCore/svg/properties/SVGAnimatedPropertyPairAccessor.h

```diff
@@ -60,8 +60,10 @@ class SVGAnimatedPropertyPairAccessor : public SVGMemberAccessor<OwnerType> {
 
     void detach(const OwnerType& owner) const override
     {
-        property1(owner)->detach();
-        property2(owner)->detach();
+        const Ref<AnimatedPropertyType1>& prop1 = property1(owner);
```

Comment:
This way we would normally fix this is by introducing a new `propertyProperty1()` getter which returns a `Ref<AnimatedPropertyType1>` and call this here instead of using an unnecessary local variable.